content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Multicluster Install
+description: Configure an Istio mesh spanning multiple Kubernetes clusters.
+weight: 30
+keywords: [kubernetes,multicluster]
+Zaliases:
+    - /docs/setup/kubernetes/multicluster-install/
+    - /docs/setup/kubernetes/multicluster/
+    - /docs/setup/kubernetes/install/multicluster/
+    - /docs/setup/install/multicluster/gateways/
+    - /docs/setup/install/multicluster/shared-vpn/
+    - /docs/setup/install/multicluster/shared-gateways/
+---
+
+This guide describes how to configure an Istio mesh that includes multiple Kubernetes clusters. We focus here
+on the details of getting it wired up, refer to [multicluster deployment model](/docs/setup/deployment-models/#multiple-clusters) for
+additional background information.
+
+We'll first show how to connect two clusters that are on the same network together, and then
+we'll show how to attach a third cluster that's on a different network.
+
+## Before you begin
+
+The procedures we describe here are primarily intended to be used with relatively pristine clusters,
+where Istio hasn't already been deployed. TBD: WHAT SHOULD WE SAY ABOUT EXISTING CLUSTER? SOL?
```

Comment:
The steps should work with existing Istio installs, but the transition may be disruptive since we need to switch the root CA in one of the clusters. There are some ways to make this smoother but the feature didn't make the 1.4 cutoff.